core/ExceptionHandler.js

```diff
@@ -50,6 +55,8 @@ export class ExceptionHandler {
      *      'isAutoRefresh' fetch exceptions.
      * @param {boolean} [options.showAlert] - display an alert dialog to the user. Default true,
      *      excepting 'isAutoRefresh' and 'isFetchAborted' exceptions.
+     * @param {boolean} [options.showWithToast] - if `showAlert`, display the error message in a toast
```

Comment:
How about alertMode= dialog|toast, defaulting to dialog?  Might be more extensible, and clearer w.r.t showAlert property.

Reply:
Great idea, will do 👍

Reply:
cool, (or alertType)